eventloop/eventloop.go

```diff
@@ -82,19 +198,21 @@ loop:
 			el.startTicker(ctx, e.tickerID)
```

Comment:
I think `ctx` can be removed from the `startTicker` method since it can be accessed via the `el.ctx` field (via the `Context()` method).

Reply:
The `ctx` could possibly also be removed from the `runTicker()` method, and move the fetching of the `Context()` into `runTicker()`. But not sure it is so important.

However, I observe that `startTicker` already holds the same lock as `Context()`, and so it could simply do:
```go
var ctx context.Context
ctx, ticker.cancel = context.WithCancel(el.ctx)
```